index.html

```diff
@@ -23,5 +25,188 @@ <h2>I’am a Software Developer.</h2>
+              <span>html</span>
+              <span>bootstrap</span>
+              <span>ruby</span>
+            </div>
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+        <div class="demo-wrap">
+          <div class="main-projects  demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <div class="lang-box" id="lang-box">
+              <span>html</span>
+              <span>bootstrap</span>
+              <span>ruby</span>
+            </div>
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+        <div class="demo-wrap">
+          <div class="main-projects  demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <div class="lang-box" id="lang-box">
+              <span>html</span>
+              <span>bootstrap</span>
+              <span>ruby</span>
+            </div>
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+        <div class="demo-wrap">
+          <div class="main-projects  demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <div class="lang-box" id="lang-box">
+              <span>html</span>
+              <span>bootstrap</span>
+              <span>ruby</span>
+            </div>
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+        <div class="demo-wrap" id="last-item">
+          <div class="main-projects demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <div class="lang-box" id="lang-box">
+              <span>html</span>
+              <span>bootstrap</span>
+              <span>ruby</span>
+            </div>
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+    <footer class="footer">
+      <h1>About me</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
- [ ] Kindly follow the proper requirements for styling the project

<img width="497" alt="Screen Shot 2022-10-29 at 05 24 54" src="https://user-images.githubusercontent.com/81444338/198831457-aa22975d-7237-45a6-86ff-b6e62025d8ac.png">

Reply:
Thanks.